client/app/account/user/contacts/update/user-contacts-update.controller.js

```diff
@@ -124,7 +124,7 @@ angular.module('UserAccount').controller('UserAccount.controllers.update', [
             $scope.controls.legalforms = legalforms;
           }),
           Contacts.getCountryEnum().then((countries) => {
-            $scope.controls.countries = countries;
+            $scope.controls.countries = _.reject(countries, country => country === 'UNKNOWN');
```

Comment:
To avoid lodash use you could : 
```suggestion
            $scope.controls.countries = countries.filter(country => country !== 'UNKNOWN') 
```

Reply:
This is done. Thanks for the suggestion :)